candle-nn/src/linear.rs

```diff
@@ -76,6 +77,7 @@ pub fn linear_no_bias(in_dim: usize, out_dim: usize, vb: crate::VarBuilder) -> R
     Ok(Linear::new(ws, None))
 }
 
+/// Create or initialize a new linear layer, bias depending on the argument.
```

Comment:
This function was the only one in the module without a line of documentation. THe function just switches between the other two based on the  `bias` boolean, so I just added this.